.storybook/rollup.config.js

```diff
@@ -0,0 +1,14 @@
+import mainConfig from '../rollup.config'
+
+let storybookComponentsPackageConfig = Object.assign({}, mainConfig)
```

Comment:
Use `const` instead of `let` ? I don't see any new assignment to `storybookComponentsPackageConfig`.

Also, this is a just a matter of style but I like to use the spread operator for object assignments. Here, `Object.assign{{}, mainconfig)` could become `{...mainConfig}`. What do you think?

Reply:
I'm not that sure here: I know that `const` only forbids the identifier to be reassigned, yet it passes the message that is object is quite "constant", i.e. immutable.

The lines just below reassigned properties of this object, so I was not that inclined to use `const`.

Reply:
I understand but still, I disagree :-). My preference would be to stick with the language semantic and maybe even enforce the eslint `prefer-const: "error"` rule. Any objection?